refinery/lib/structures.py

```diff
@@ -376,7 +376,7 @@ def read_exactly(self, size: Optional[int] = None, peek: bool = False) -> T:
         """
         Read bytes from the underlying stream. Raises a `RuntimeError` when the stream is not currently
         byte-aligned, i.e. when `refinery.lib.structures.StructReader.byte_aligned` is `False`. Raises
-        an exception of type `refinery.lib.structures.EOF` when less data is available in the stream than
+        an exception of type `refinery.lib.structures.EOF` when fewer data is available in the stream than
```

Comment:
With this one I disagree =)

Reply:
You said in your comment that data is not countable and therefore, "less" should be used over "fewer". This proposed change, however, is doing the opposite. On the other hand, it is strictly so that "data" is the plural of "datum". This led me down [a deep rabbit hole](https://en.wikipedia.org/wiki/Data_(word)) to finally arrive at the conclusion that I don't really care.